pkg/model/sdk_api.go

```diff
@@ -14,177 +14,32 @@
 package model
 
 import (
-	"errors"
-	"fmt"
-	"io/ioutil"
-	"os"
-	"path/filepath"
-	"sort"
 	"strings"
 
-	"gopkg.in/src-d/go-git.v4"
-
 	ackgenconfig "github.com/aws-controllers-k8s/code-generator/pkg/generate/config"
 	"github.com/aws-controllers-k8s/code-generator/pkg/names"
 	"github.com/aws-controllers-k8s/code-generator/pkg/util"
-
```

Comment:
ditto

Reply:
Generally we do:

```go
import (
   <std packages>

   <3rd party packages>

   <local packages>
)
```

so @A-Hilaly is saying the above should be:

```go
import (
	"strings"

	awssdkmodel "github.com/aws/aws-sdk-go/private/model/api"

	ackgenconfig "github.com/aws-controllers-k8s/code-generator/pkg/generate/config"
	"github.com/aws-controllers-k8s/code-generator/pkg/names"
	"github.com/aws-controllers-k8s/code-generator/pkg/util"
)
```